roles/app/templates/application.yml.j2

```diff
@@ -21,10 +21,6 @@ GOOGLE_MAPS_API_KEY: {{ google_maps_api_key }}
 ADMIN_EMAIL: {{ admin_email }}
 ADMIN_PASSWORD: {{ admin_password }}
 
-{% if enable_rails_apm is defined %}
-DATADOG_RAILS_APM: {{ enable_rails_apm }}
-{% endif %}
-
```

Comment:
I am confused as why this duplicated in `roles/app/templates/env.j2` (or maybe it's the other way around ?)

Reply:
We used to have our configuration in `config/application.yml` and switched to `.env` at some point. So in this transition it's duplicated. But I reckon that we can remove application.yml now. All servers should only be used .env now.